modules/CmpdRegAdmin/src/server/routes/CmpdRegAdminRoutes.coffee

```diff
@@ -13,11 +13,12 @@ exports.setupRoutes = (app, loginRoutes) ->
 	app.get '/api/cmpdRegAdmin/:entityType', loginRoutes.ensureAuthenticated, exports.getCmpdRegEntities
 	app.get '/api/cmpdRegAdmin/:entityType/search/:searchTerm', loginRoutes.ensureAuthenticated, exports.searchCmpdRegEntities
 	app.post '/api/cmpdRegAdmin/:entityType/validateBeforeSave', loginRoutes.ensureAuthenticated, loginRoutes.ensureCmpdRegAdmin, exports.validateCmpdRegEntityBeforeSave
-
 	app.get '/api/cmpdRegAdmin/:entityType/:id', loginRoutes.ensureAuthenticated, exports.getCmpdRegEntityById
 	app.post '/api/cmpdRegAdmin/:entityType', loginRoutes.ensureAuthenticated, loginRoutes.ensureCmpdRegAdmin, exports.saveCmpdRegEntity
 	app.put '/api/cmpdRegAdmin/:entityType/:id', loginRoutes.ensureAuthenticated, loginRoutes.ensureCmpdRegAdmin, exports.updateCmpdRegEntity
 	app.delete '/api/cmpdRegAdmin/:entityType/:id', loginRoutes.ensureAuthenticated, loginRoutes.ensureCmpdRegAdmin, exports.deleteCmpdRegEntity
+	app.post '/api/cmpdRegAdmin/lotServices/reparent/lot', loginRoutes.ensureAuthenticated, loginRoutes.ensureCmpdRegAdmin, exports.reparentLot
```

Comment:
Just an observation: I find it interesting that you moved these routes over from `CmpdRegRoutes` to `CmpdRegAdminRoutes`. My understanding of the CmpdRegAdminRoutes were modules in the ACAS UI that revolved around administration of CmpdReg things, whereas CmpdRegRoutes were for parts of the CReg UI, whether or not they require user or admin level permissions.

In this case it's a route backing functionality in the CReg UI, but it seems like you moved it because it requires CReg Admin level permissions. Where the code sits doesn't make a big difference though, so it's fine either way.

Reply:
I did move it because it had the the `loginRoutes.ensureCmpdRegAdmin` middleware method.  I get your point but leaving this as is for now.